src/py42/sdk/queries/__init__.py

```diff
@@ -26,3 +27,16 @@ def any(cls, *args):
     @classmethod
     def all(cls, *args):
         return cls(*args)
+
+
+class Choices:
+    """Helper class to provide the choices() method to a class."""
```

Comment:
If you put back ticks, e.g. ``choices()`` around the method name, it will look nicer in the generated-documentation page.